libs/wingsdk/test/shared-gcp/bucket.inflight.test.ts

```diff
@@ -310,5 +310,27 @@ test("tryDelete a non-existent object from the bucket", async () => {
   expect(res).toBe(false);
 });
 
+/**
+ * Not usable since `mock-gcs` module misses `File.copy()`
+ */
+// test("copy objects within the bucket", async () => {
+//   // GIVEN
+//   const BUCKET_NAME = "BUCKET_NAME";
+//   const SRC_KEY = "SRC/KEY";
+//   const DST_KEY = "DST/KEY";
+//   const SRC_VALUE = "hello world";
+//   const mockStorage = new MockStorage();
+//   await mockStorage.bucket(BUCKET_NAME).file(SRC_KEY).save(SRC_VALUE);
+
+//   // WHEN
+//   const client = new BucketClient(BUCKET_NAME, mockStorage as any);
+//   const response1 = await client.copy(SRC_KEY, SRC_KEY);
+//   const response2 = await client.copy(SRC_KEY, DST_KEY);
+
+//   // THEN
+//   expect(response1).toEqual(undefined);
+//   expect(response2).toEqual(undefined);
+// });
```

Comment:
Do you need help in overcoming this? 
what about spying on the `client.bucket.file.copy` method and check if the inner gcp command is called with the right parameters, then mocking the resolved value correctly?

Reply:
How do you spy on a method that doesn't exist?

Reply:
does `client.bucket.file` exist? 
if so, I guess you can do `client.bucket.file.copy = vi.fn`
then spying on it :)